pom.xml

```diff
@@ -62,8 +62,33 @@
         <dependency>
             <groupId>software.amazon.awssdk</groupId>
             <artifactId>kms</artifactId>
-            <version>2.18.38</version>
+            <version>2.20.91</version>
+        </dependency>
+
+        <dependency>
+            <groupId>software.amazon.cryptography</groupId>
+            <artifactId>aws-cryptographic-material-providers</artifactId>
+            <version>1.0.2</version>
+        </dependency>
+
+        <dependency>
+            <groupId>software.amazon.cryptography</groupId>
+            <artifactId>TestAwsCryptographicMaterialProviders</artifactId>
+            <version>1.0-SNAPSHOT</version>
             <optional>true</optional>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>software.amazon.smithy.dafny</groupId>
+            <artifactId>conversion</artifactId>
+            <version>0.1</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.dafny</groupId>
+            <artifactId>DafnyRuntime</artifactId>
+            <version>4.2.0</version>
```

Comment:
why do we need these?
Does the ESDK-Java actually have a hard dependency on these?

Reply:
We need those dependencies to build the test vectors mpl jar for testing, so I'll change the test dependency to optional.